### DISCUSSION TITLE
Will Agents be supported in Java?

### DISCUSSION THREAD

**Discussion by roberst00** [microsoft/semantic-kernel#7303] [General]
Agents are a helpful abstraction, especially the multi-agent conversations. Will these be supported in the Java implementation? And if so, when is that expected to happen?

**Comment by dsgrieve** [comment]
I started implementing assistants a while ago but shelved the work due to other priorities. When this work will resume is to be determined.

Here is the issue: https://github.com/microsoft/semantic-kernel-java/issues/19

Our backlog is in the [Semantic Kernel project](https://github.com/orgs/microsoft/projects/866/views/3?filterQuery=-status%3A%22No+Status%22%2CReleased%2C%22Backlog*%22%2CEpic%2C%22V1+Drafts%22++label%3A+java) on github.
